plugins/org.eclipse.fordiac.ide.library/src/org/eclipse/fordiac/ide/library/messages.properties

```diff
@@ -16,7 +16,7 @@ Confirm=Confirm
 
 ErrorMarkerLibNotAvailable=Library not available: {0} Version: {1}
```

Comment:
can you add the path to the manifest that caused the error.
this is confusing otherwise, especially when the error is caused by a transitive import